src/pages/chainlink-nodes/v1/using-forwarder.md

```diff
@@ -0,0 +1,323 @@
+:::
+
+## Test the transaction-sending strategy
+
+### Create API Requests
+
+1. Open [APIConsumerForwarder.sol](https://remix.ethereum.org/#url=https://docs.chain.link/samples/APIRequests/APIConsumerForwarder.sol) in RemixIDE.
+
+1. Note that `setChainlinkToken(0x326C977E6efc84E512bB9C30f76E30c160eD06FB)` is configured for _Goerli_.
+
+1. On the **Compiler** tab, click the **Compile** button for `APIConsumerForwarder.sol`.
+
+1. On the **Deploy and Run** tab, configure the following settings:
+
+   - Select _Injected Provider_ as your environment. Make sure your metamask is connected to Goerli.
+   - Select _APIConsumerForwarder_ from the **Contract** menu.
+
+1. Click **Deploy**. MetaMask prompts you to confirm the transaction.
+
+1. Fund the contract by sending LINK to the contract's address. See the [Fund your contract](/resources/fund-your-contract/) page for instructions. The address for the `ATestnetConsumer` contract is on the list of your deployed contracts in Remix. You can fund your contract with 1 LINK.
+
+1. After you fund the contract, create a request. Input your operator contract address and the job ID for the `Get > Uint256` job into the `requestEthereumPrice` request method **without dashes**. The job ID is the `externalJobID` parameter, which you can find on your job's definition page in the Node Operators UI.
+
+1. Click the **transact** button for the `requestEthereumPrice` function and approve the transaction in Metamask. The `requestEthereumPrice` function asks the node to retrieve `uint256` data specifically from [https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD](https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD).
+
+1. After the transaction processes, you can see the details for the complete the job run the **Runs** page in the Node Operators UI.
+
+1. In Remix, click the `currentPrice` variable to see the current price updated on your consumer contract.
+
+1. Input your operator contract address and the job ID for the `Get > String` job into the `requestFirstId` request method **without dashes**. The job ID is the `externalJobID` parameter, which you can find on your job's definition page in the Node Operators UI.
+
+1. Click the **transact** button for the `requestFirstId` function and approve the transaction in Metamask. The `requestFirstId` function asks the node to retrieve the first `id` from [https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&per_page=10](https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&per_page=10).
+
+1. After the transaction processes, you can see the details for the complete the job run the **Runs** page in the Node Operators UI.
+
+1. In Remix, click the `id` variable to see the current price updated on your consumer contract.
+
+### Check Forwarder
```

Comment:
This section still TBD?

Reply:
@dwightjl yes , waiting for the next docker release